docs/docs/sourcing-from-seams-cms.md

```diff
@@ -0,0 +1,197 @@
+---
+
+## Overview
+
+[Seams-CMS](https://seams-cms.com) is a headless CMS system that allows you to easily manage content and data with either simple or complex structures and supports localization.
+
+Creating a Seams-CMS account is free for hobby or small sites. For this tutorial, we will be using the demo workspace that is automatically generated when you create a new account. Make sure you add an API key to this workspace in the API key settings. We will be needing your workspace ID and the (read-only) API key shortly.
+
+## Setup
+
+Create a new Gatsby site with the [default starter](https://github.com/gatsbyjs/gatsby-starter-default):
+
+Run this in your terminal:
+
+```shell
+gatsby new seamscms-site
+cd seamscms-site
+```
+
+### Install the source plugin
+
+Next, we will be installing the Seams-CMS Gatsby source plugin:
+
+```shell
+npm install --save  seams-cms-gatsby-source
+```
+
+### Adding configuration
+
+When you have installed the plugin, it's time to configure it. Add the following code to the `gatsby-config.js` file found in the root of your project.
+
+```javascript:title=gatsby-config.js
+module.exports = {
+  ...
+  plugins: [
+    ...
+    {
+      // Seams-CMS source plug
+      resolve: 'seams-cms-gatsby-source',
+      options: {
+        workspace: <WORKSPACE_ID>,
+        apiKey: <API_KEY>,
+        contentTypes: ['blogpost'],
+      },
+    }
+    ...
+  ],
+  ...
+}
+```
+
+You have to change `<WORKSPACE_ID>` and `<API_KEY>` into the ID of the workspace from your own Seams-CMS account and the newly generated API key inside that workspace. Within the `contentTypes` array, we define all the ID's of the content types we like to import from Seams-CMS into Gatsby, in this case we only want to import the `blogpost` content.
+
+### Run and check our connection
+
+Next, we can run Gatsby and let it fetch our blogposts from Seams-CMS:
+
+```shell
+gatsby develop
+```
+
+To check if it worked, we can log into the GraphQL admin at `http://localhost:8000/___graphql` and look for an `allSeamsCmsBlogpost` section. If found, it means that blogposts are correctly fetched and stored in the GraphQL layer of Gatsby, ready to be consumed.
+
+To see what kind of data it holds, try running the following query:
+
+```graphql
+query MyQuery {
+  allSeamsCmsBlogpost {
+    edges {
+      node {
+        content {
+          title {
+            value
+          }
+          content {
+            value
+          }
+          author {
+            value {
+              content {
+                name {
+                  value
+                }
+              }
+            }
+          }
+          categories {
+            value {
+              content {
+                name {
+                  value
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+This will return the blogposts title, content, author name and category name in which the blogpost was posted.
```

Comment:
```suggestion
This will return the title, content, author name, and categories for each blog post.
```

The [Oxford comma](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#make-lists-clear-with-the-oxford-comma) can help clarify lists.